### PR TITLE
Enhance Integration tests for the error end event

### PR DESCRIPTION
**Pull request by robpal94**
## What did you change?
This PR extends the test process for the ErrorEndEvent with boundary events, that tries to catch an unrelated and an anonymous error. 

Both cases should never occur, since the ErrorEndEvent should throw a specified error.

## How can others test the changes?
This PR can only be testes, if the handlers for the ErrorEndEvent are merged. 
See https://github.com/process-engine/process_engine/pull/102

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: _integration_tests/test/error_end_event_test.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ describe.skip('Error End Event - ', () => {
     should(processInstancePromise).be.rejectedWith(expectedErrorObject);
   });
 
-  it('should execute a call activity which ends with an error boundary event', async () => {
+  it.skip('should execute a call activity which ends with an error boundary event', async () => {
```
Comment (ElRaptorus) [inline _integration_tests/test/error_end_event_test.js]: Intentional, or overlooked?

Comment (robpal94) [reply _integration_tests/test/error_end_event_test.js]: Intentional. 

I talked with @5ebastianMeier , that we can skip the tests which uses a call activity for now.

Comment (ElRaptorus) [reply _integration_tests/test/error_end_event_test.js]: Makes sense. Thx.

**Comment by robpal94** [comment]
I will hold this commit back until the ProcessEngine that supports ErrorEndEvents is published.

**Pull request merged**